### ISSUE TITLE
Issue with disconnected install when attempting to parse package repo URL

### ISSUE THREAD

**Issue by swade1987** [apprenda/kismatic#749]
`kismatic-cluster.yaml` setup below ...

```
cluster:
  name: kubernetes
  admin_password: HoVmiBeJrudasvi19      # This password is used to login to the Kubernetes Dashboard and can also be used for administration without a security certificate.
  disable_package_installation: false    # When true, installation will not occur if any node is missing the correct deb/rpm packages. When false, the installer will attempt to install missing packages for you.
  package_repository_urls: "http://138.68.162.254"            # Comma-separated list of URLs of the repositories that should be used during installation. These repositories must contain the kismatic packages and all their transitive dependencies.
  disconnected_installation: true       # Set to true if you have already installed the required packages on the nodes or provided a local URL in package_repository_urls containing those packages.
  disable_registry_seeding: false 
```

See output below ..

```
Configure Kismatic Package Repos                                                [ERROR]
- Task: clean kismatic yum cache
  storage2:                                                                     [ERROR]
---- STDOUT ----
Loaded plugins: fastestmirror
Determining fastest mirrors
---- STDERR ----
Repository 'kismatic-0': Error parsing config: Error parsing "baseurl = "package_repository_urls.split(',')"": URL must be http, ftp, file or https not ""
There are no enabled repos.
 Run "yum repolist all" to see the repos you have.
 You can enable repos with yum-config-manager --enable <repo>
```

**Comment by dkoshkin** [comment]
Thanks @swade1987 looks like the `package_repository_urls` variable is not being expanded correctly.

**Comment by dkoshkin** [comment]
Closed with #751
